assets/extensions/update-notification/update-available-label.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Update available label component.
+ *
+ * @param {Object} props              Component props.
+ * @param {number} props.updatesCount Number of extension updates.
+ */
+const updateAvailableLabel = ( { updatesCount } ) => (
```

Comment:
Good approach separating this! =)

I just wonder if would be better if we brought the `small` wrapper, and the icon together.
And we should just capitalize the first letter here.

Reply:
Fo sho - https://github.com/Automattic/sensei/pull/4213/commits/dcb1bf28ac538c6ad2dcb95436639f62ba50e75a